### ISSUE TITLE
Is target-driven visual navigation supported?

### ISSUE THREAD

**Issue by playerkk** [allenai/ai2thor#25]
I was wondering if the mechanism of target-driven visual navigation described in the ICRA 2017 paper supported. Thanks.

**Comment by roozbehm** [comment]
Do you mean whether you can implement that using our new framework?

**Comment by playerkk** [comment]
Yes. Thanks.

**Comment by roozbehm** [comment]
Yes, you can implement the same algorithms using this framework. We might provide the code for that within a few weeks.

**Comment by playerkk** [comment]
I didn't see any interface to set the target object, like env.initialize_target(target) in the THOR-Challenge release. Could you point me where this is? Thanks.

**Comment by roozbehm** [comment]
That was specific to the THOR challenge. You can re-use the target images and re-implement it. It shouldn't be very difficult.

**Comment by playerkk** [comment]
Thanks. Will close the issue.

**Comment by roozbehm** [comment]
We randomly initialize the scenes so the egg will not be at that location.

**Comment by YunlianMoon** [comment]
Hi, do you know how to generate scene dumps like the ICRA 2017 paper?
